src/codemagic/tools/_app_store_connect/action_groups/builds_action_group.py

```diff
@@ -150,6 +194,7 @@ def add_beta_test_info(
     def submit_to_testflight(
             self,
             build_id: ResourceId,
+            expire_build_submitted_for_review: bool = False,
             max_build_processing_wait: Optional[Union[int, Types.MaxBuildProcessingWait]] = None,
```

Comment:
For backwards compatibility it is better to add new method arguments to the end of method signature.
```suggestion
            max_build_processing_wait: Optional[Union[int, Types.MaxBuildProcessingWait]] = None,
            expire_build_submitted_for_review: bool = False,
```